Aztec/Classes/Libxml2/DOM/ElementNode.swift

```diff
@@ -91,7 +91,13 @@ extension Libxml2 {
         /// - Returns: true if the last child of this element is a block level element, false otherwise
         ///
         func isLastChildBlockLevelElement() -> Bool {
-            if let lastChild = children.last as? ElementNode {
+            let childrenFiltered = children.filter { (node) -> Bool in
```

Comment:
This may become slightly more readable using a reversed loop... something like this:

```swift
for child in children.reversed() {
    if let elementNode = node as? ElementNode {
        return elementNode.isBlockLevelElement()
    } else if let textNode = node as? TextNode where textNode.text().isEmpty {
        continue
    } else {
        return false
    }
}
```